app/services/adapters/structs/crime_application.rb

```diff
@@ -16,6 +16,8 @@ def applicant
       end
 
       def case
+        return if pse?
```

Comment:
Again, unrelated, but PSE has no case, and the CaseDetails struct tries to rebuild case_type in case it's an Appeal. So we can skip this